modules/quick-starts-accessing-and-executing-code-snippets.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * web_console/creating-quick-start-tutorials.adoc
+
+[id="quick-starts-accessing-and-executing-code-snippets_{context}"]
+= Code snippet markdown reference
+
+You can execute a CLI code snippet when it is included in a quick start from the web console. To use this feature, you must first install the Web Terminal Operator. The web terminal and code snippet actions that execute in the web terminal are not present if you do not install the Web Terminal Operator. Alternatively, you can copy a code snippet to the clipboard regardless of whether you have the Web Terminal Operator installed or not.
+
+[id="quick-starts-syntax-for-inline-code-snippets_{context}"]
+== Syntax for inline code snippets
+
+[source,yaml]
+----
+`code block`{{copy}}
+`code block`{{execute}}
```

Comment:
> `code block`{{execute}}

Just FYI, execute syntax has `Copy to Clipboard` action present implicitly i.e. if execute syntax is used copy to clipboard action will be present regardless of web terminal operator.

Reply:
@sahil143 Thanks! I added a note